scripts/swift-analyzer.py

```
@@ -0,0 +1,208 @@
+import os
+import requests
+import json
+import subprocess
```

Ruff (0.11.9)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)